src/main/java/frc/team2412/robot/subsystems/DrivebaseSubsystem.java

```diff
@@ -147,22 +159,39 @@ public DrivebaseSubsystem() {
 	 * @param rotation
 	 * @param fieldOriented
 	 */
-	public void drive(double forward, double strafe, Rotation2d rotation, boolean fieldOriented) {
-		SwerveModuleState[] moduleStates = getModuleStates(new ChassisSpeeds(0, 0, 0));
+	public void drive(
+			double forward,
+			double strafe,
+			Rotation2d rotation,
+			boolean fieldOriented,
+			boolean autoBalance) {
+		// Auto balancing will only be used in autonomous
+		if (autoBalance) {
```

Comment:
How will the robot stop from driving off the charge station if another robot partially tips the ramp?
How will the robot handle the possibility of running into the charge station when the other side is down and we are facing a 71.5 degree slope (I am assuming digging our wheels into the carpet will be frowned upon again this year.)?